dts/bindings/interrupt-controller/arm,gic.yaml

```diff
@@ -18,6 +18,7 @@ properties:
       required: true
 
 interrupt-cells:
+  - type
   - irq
   - priority
```

Comment:
Can we swap `-flags` and `-priority` order.

Reply:
Sounds fair.